lib/config.js

```diff
@@ -216,8 +216,62 @@ function parseConfig(config) {
   )
 }
 
+/**
+ * Decodes a base64 encoded string into a config
+ *
+ * @param {string} encodedRuleSet A base64 encoded string that needs decoding
+ * @returns {Object} The loaded repolinter json config
+ * @throws Will throw an error if unable to parse config or if config is invalid
+ */
+ async function decodeConfig(encodedRuleSet) {
```

Comment:
I wonder if we could refactor the existing loadConfig function, so that we don't end up duplicating all of this?

(I know you mentioned this wasn't ready for review, so maybe this is what you're planning on doing anyway.  I just happen to run across this, and figured I'd go ahead and nudge in that direction)

Reply:
Good suggestion @willnorris, 

I will see what I can do tomorrow. Would Repolinter be interested in a feature addition like this?

Reply:
Certainly, especially if Philip's is using it.  I don't have a use-case for it myself, but you're using it then let's add it.  Especially since I think that with a little refactoring we can reduce this to just a handful of lines to decode the string, and then pass it into the normal config processing logic.

Reply:
That sounds good. I'm no JavaScript expert so I think we can improve the code significantly indeed.
Do you all use any other platform (like Slack), where we can discuss PRs, feature requests, etc? We might have some other features that might be interesting.

Reply:
Yep, in the #topic-tools channel of https://slack.todogroup.org/

Reply:
Great, didn't know about that channel. Will reach out tomorrow. Appreciate the comments! Keep up the good work 🦾